src/west/_bootstrap/main.py

```diff
@@ -207,12 +209,17 @@ def bootstrap(args):
 
     # Clone the west source code and the manifest into west/. Git will create
     # the west/ directory if it does not exist.
+    tempdir = tempfile.gettempdir()
 
-    clone('manifest repository', manifest_url, manifest_rev,
-          os.path.join(directory, MANIFEST))
+    i = 0
+    while os.path.exists(os.path.join(tempdir, 'manifest_{}'.format(str(i)))):
+        i = i + 1
+    manifest_temp = os.path.join(tempdir, 'manifest_{}'.format(str(i)))
+
+    clone('manifest repository', manifest_url, manifest_rev, manifest_temp)
```

Comment:
It might be worth deleting the temporary directory if the rest of the steps fail.

Reply:
at later stage maybe.
Keeping the temporary folder in case something goes wrong may help the user / developer to investigate what happened. E.g invalid data in manifest, config file, other ?

Same reason the folder is also not removed when `init` completes.
After all it is placed in the system temp folder.

@carlescufi thoughts ?

Reply:
I would delete it anyway because the repo is big enough to start causing trouble with people's temp folders getting big and the debugging of what went wrong can be done by looking directly at the original repo's code

Reply:
temporary directory now removed when int completes (regardless of success or failure)